weblate/machinery/tests.py

```diff
@@ -904,6 +934,39 @@ def test_bug(self):
             self.assert_translate(self.SUPPORTED, self.SOURCE_TRANSLATED, 0)
 
 
+class SystranTranslationTest(BaseMachineTranslationTest):
+    MACHINE_CLS = SystranTranslation
+    EXPECTED_LEN = 1
+    CONFIGURATION = {
+        "key": "key",
+    }
+
+    def mock_empty(self):
+        raise SkipTest("Not tested")
+
+    def mock_error(self):
+        raise SkipTest("Not tested")
+
+    def mock_response(self):
+        responses.add(
+            responses.GET,
+            "https://api-translate.systran.net/translation/apiVersion",
+            json={"version": "2.11.0"},
+        )
+
+        responses.add(
+            responses.GET,
+            "https://api-translate.systran.net/translation/supportedLanguages",
+            json=SYSTRAN_LANGUAGE_JSON,
+        )
+
+        responses.add(
+            responses.GET,
```

Comment:
POST is used here (see https://github.com/WeblateOrg/weblate/pull/10955/files#diff-21a569b0906c395b0a03c3c2d6499eac7101930e86d70ef9a5df9937a66e6899R43):

```suggestion
            responses.POST,
```